config/locales/en.yml

```diff
@@ -899,6 +903,9 @@ en:
     progress_for_clearance: The Disclosure team has been notified this case is ready for clearance
     case/ico:
       case_cleared: "The response has been cleared and is ready to be sent to the ICO"
+      case_required_further_action_same_responder: "The responding team has been notified and case is ready for redrafting."
```

Comment:
Not a blocker but we could avoid a little repetition here with name-spacing on the common parts:

```
required:
    further_action:
         same_responder: etc...
         same_team: etc...
         reassign: etc...
```